src/main/kotlin/ok/work/etoroapi/client/EtoroHttpClient.kt

```diff
@@ -116,7 +117,12 @@ class EtoroHttpClient {
                     if (!imageData.has("Uri")) {
                         continue;
                     }
-                    val image = Image(imageData.getInt("Width"), imageData.getInt("Height"), imageData.getString("Uri"))
+                    val image = Image(0,0, "");
+                    try {
+                        val image = Image(imageData.getInt("Width"), imageData.getInt("Height"), imageData.getString("Uri"))
```

Comment:
You need to add a required boolean parameter to `CreateSessionRequest` class constructor, otherwise the request silently crashes. However, at least for me, the updates still don't occur even after

Reply:
`class CreateSessionRequest(targetServer: String, polling: Boolean, cause: String?, options: InternalConnectionOptions, details: InternalConnectionDetails, delay: Long, password: String?, oldSession: String?, serverBusy: Boolean) : SessionRequest(polling, delay) {`